my-app/src/App.tsx

```diff
@@ -13,15 +13,15 @@ const theme = createTheme({
         sm: 480,
         md: 767,
         lg: 1200,
-        xl: 1519,
+        xl: 1920,
       },
 
       spacing: { 
-        xs: 0,
-        sm: 1.8,
-        md: 5,
-        lg: 10.25,
-        xl: 12.5,
+        xs: 15px,
+        sm: 38px,
+        md: 40px,
+        lg: 82px,
+        xl: 100px,
```

Comment:
Again this spacing is wrong, you can't have 38 than 40, that doesn't make sense. Keep one only. Than 82px and 100px are 100% not correct. But let's  keep this for later.